src/groups/special_linear.jl

```diff
@@ -153,9 +153,4 @@ function Base.show(io::IO, M::SpecialLinear{Tuple{Int},𝔽}) where {𝔽}
     return print(io, "SpecialLinear($n, $(𝔽); parameter=:field)")
 end
 
-translate_diff(::SpecialLinear, p, q, X, ::LeftForwardAction) = X
-translate_diff(::SpecialLinear, p, q, X, ::RightBackwardAction) = p \ X * p
-
-function translate_diff!(G::SpecialLinear, Y, p, q, X, conv::ActionDirectionAndSide)
-    return copyto!(Y, translate_diff(G, p, q, X, conv))
-end
+adjoint_action!(G::SpecialLinear, Y, p, q, X, conv::LeftAction) = p \ X * p
```

Comment:
I don't think this method looks correct? There are two `adjoint_action!` methods for `SpecialLinear` above so it's probably not needed.

Reply:
This method doesn't make any sense: wrong number of arguments, not using the pre-allocated `Y`, it's all wrong.